firefox-ios/firefox-ios-tests/Tests/ClientTests/ContentBlockerTests.swift

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Client
+
+final class ContentBlockerTests: XCTestCase {
+    func testCompileListsNotInStore_callsCompletionHandlerSuccessfully() {
+        ensureAllRulesAreRemovedFromStore()
+        let expectation = XCTestExpectation()
+        ContentBlocker.shared.compileListsNotInStore {
+            expectation.fulfill()
+        }
+        wait(for: [expectation])
+    }
+
+    private func ensureAllRulesAreRemovedFromStore() {
```

Comment:
Feels this should be in a `setUp` method instead of being called at the top of the test func?

Reply:
yes true, i'll move it there